VK/VK/Presentation/Friends/Models/VKItemResponse.swift

```diff
@@ -0,0 +1,14 @@
+// VKItemResponse.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Элементы ответа ВК
+struct VKItemResponse: Decodable {
+    var items: [Friend]
```

Comment:
```suggestion
    var friends: [Friend]
```
мне Женя так поправлял( убирал все items и model
типо должны совпадать имена